frontends/infinite-corridor/src/pages/SearchPage.tsx

```diff
@@ -92,51 +107,66 @@ const SearchPage: React.FC = () => {
     )
 
   return (
-    <div>
-      <BannerPageWrapper>
-        <BannerPageHeader tall compactOnMobile>
-          <Searchbox
-            className="course-searchbox"
-            onChange={updateText}
-            value={text || ""}
-            onClear={clearText}
-            //@ts-expect-error - types need to be fixed in course-search-utils
-            onSubmit={onSubmit}
-            autoFocus
-          />
-        </BannerPageHeader>
-      </BannerPageWrapper>
-      <InfiniteScroll
-        hasMore={from + pageSize < total}
-        loadMore={loadMore}
-        initialLoad={from === 0}
-        loader={<div className="loader">Loading ...</div>}
-      >
-        <section>
-          {searchApiFailed ? (
-            <div className="no-results-found">
-              <span>Oops! Something went wrong.</span>
-            </div>
-          ) : completedInitialLoad ? (
-            results.length === 0 ? (
-              <div className="no-results-found">
-                <span>No results found for your query</span>
-              </div>
-            ) : (
-              results.map((hit, idx) => (
-                <Cell key={idx} width={SEARCH_RESUTLT_CELL_WIDTH}>
-                  <LearningResourceCard
-                    object={searchResultToLearningResource(hit._source)}
-                  />
-                </Cell>
-              ))
-            )
-          ) : (
-            <span>Loading...</span>
-          )}
-        </section>
-      </InfiniteScroll>
-    </div>
+    <BannerPage
+      omitBackground
+      bannerContent={
+        <Searchbox
+          className="course-searchbox"
+          onChange={updateText}
+          value={text || ""}
+          onClear={clearText}
+          //@ts-expect-error - types need to be fixed in course-search-utils
+          onSubmit={onSubmit}
+          autoFocus
+        />
+      }
+    >
+      <Container disableGutters>
+        <Grid container>
+          <Grid item xs={3}>
```

Comment:
`<Grid container />` and `<Grid item />` are like our old Grid and Cell components, but not custom.

`xs={3}` says "Use 3 of the 12 columns on any screen xs or larger" (so, all screens)